akka-testkit-typed/src/test/scala/akka/typed/testkit/EffectfulActorContextSpec.scala

```diff
@@ -80,7 +80,7 @@ object EffectfulActorContextSpec {
 
 class EffectfulActorContextSpec extends FlatSpec with Matchers {
 
-  private val props = Props.empty.withMailboxCapacity(10)
```

Comment:
was this not working on the adapted one? Does this mean we lose the concept of dropping or how it was called there (the mailbox full, it would not be dead letter but that other thing)?

Reply:
Nothing read it after i removed ActorSystemImpl so i guess not

Reply:
Bounded mailbox is not a new concept. We could make it default and/or make it simple to specify via Props, but that should be a separate ticket.

Reply:
Yeap, the new part in Typed was that there was a difference between dropped and deadletters AFAIR which I really liked. Ticket pls :)